hawc/apps/lit/migrations/0016_doi_lowercase.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+
+from django.db import migrations
+
+
+def set_doi_lowercase(apps, schema_editor):
```

Comment:
does this work or something similar?

```python
from django.db.models import F
from django.db.models.functions import Lower

Identifiers.objects.filter(database=4).update(unique_id =F(Lower('unique_id'))
```

That way it's done in the db

Reply:
I'll give that a try, I'm not too familiar with the F function yet so I didn't think it would be possible to do that in the update function